app/onboarding.tsx

```diff
@@ -124,7 +124,15 @@ const OnboardingScreen = () => {
             accessibilityLabel={t('goBack')}
             name="arrow-back"
           />
-        ) : null}
+        ) : (
+          <View />
```

Comment:
?

Reply:
100% :D there are 2 ways to do it, one is class and the second is empty view :) this was a bit easier in my opinion